client/styles/pages/view_proposal/index.scss

```diff
@@ -537,14 +537,16 @@ $discussion-title-font-size: 1.45rem;
             flex: 6;
             display: flex;
             flex-direction: row;
-            align-items: center;
+            align-items: left;
```

Comment:
Someone misnamed the parent class, so this selector doesn't actually match anything. I can push a fix with new naming scheme